composer.json

```diff
@@ -19,5 +19,12 @@
             "Devitek\\": "src/Devitek"
         }
     },
+    "extra": {
```

Comment:
Don't know whether this has an impact, since the package must be overridden manually.